<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeToolWidget</class>
 <widget class="QWidget" name="ShapeToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>116</width>
    <height>132</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QGroupBox" name="group_box">
     <property name="title">
      <string>Create</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="check_group">
        <property name="text">
         <string>Group</string>
        </property>
        <property name="icon">
         <iconset theme="object-group">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_fill">
        <property name="text">
         <string>Fill</string>
        </property>
        <property name="icon">
         <iconset theme="format-fill-color">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_stroke">
        <property name="text">
         <string>Stroke</string>
        </property>
        <property name="icon">
         <iconset theme="format-stroke-color">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_group</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_fill</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_stroke</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check_checks()</slot>
 </slots>
</ui>
